mailet/mailetdocs-maven-plugin/src/main/java/org/apache/james/mailet/MailetMatcherDescriptor.java

```diff
@@ -105,10 +110,36 @@ public void setType(Type type) {
         this.type = type;
     }
 
+    public boolean isExperimental() {
+        return experimental;
+    }
+
+    public void setExperimental(boolean experimental) {
+        this.experimental = experimental;
+    }
+
     @Override
     public String toString() {
-        return "MailetMatcherDescriptor [fullyQualifiedClassName="
-                + fullyQualifiedClassName + ", name=" + name + ", type=" + type
-                + "]";
+        return MoreObjects.toStringHelper(MailetMatcherDescriptor.class)
+                .add("fullyQualifiedClassName", fullyQualifiedClassName)
+                .add("name", name)
+                .add("info", info)
+                .add("type", type)
+                .add("experimental", experimental)
+                .toString();
     }
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
This equals method is used only for testing purpose, we don't need an hashCode method yet.